cni/network/mutlitenancy.go

```diff
@@ -214,7 +215,7 @@ func GetMultiTenancyCNIResult(
 	ifName string) (*cniTypesCurr.Result, *cns.GetNetworkContainerResponse, net.IPNet, *cniTypesCurr.Result, error) {
 
 	if nwCfg.MultiTenancy {
-		result, cnsNetworkConfig, subnetPrefix, err := getContainerNetworkConfiguration(nwCfg, "", k8sPodName, k8sNamespace, ifName)
+		result, cnsNetworkConfig, subnetPrefix, err := getContainerNetworkConfiguration(nwCfg, nwCfg.CNSUrl, k8sPodName, k8sNamespace, ifName)
```

Comment:
Already passing nwCfg as an arg, so need to pass nwCfg.CNSUrl separately

Reply:
was trying to keep the target method generic so that it can be reused in case the CNS url is not coming from CNI config... 
func getContainerNetworkConfiguration(nwCfg *cni.NetworkConfig, address string, podName string, podNamespace string, ifName string)